packages/server-wallet/src/chain-service/__chain-test__/chain-service.test.ts

```diff
@@ -26,9 +25,9 @@ const erc20AssetHolderAddress = makeAddress(process.env.ERC20_ASSET_HOLDER_ADDRE
 const erc20Address = makeAddress(process.env.ERC20_ADDRESS!);
 /* eslint-enable no-process-env, @typescript-eslint/no-non-null-assertion */
 
-if (!defaultTestConfig.rpcEndpoint) throw new Error('rpc endpoint must be defined');
-const rpcEndpoint = defaultTestConfig.rpcEndpoint;
-const provider: providers.JsonRpcProvider = new providers.JsonRpcProvider(rpcEndpoint);
+const GANACHE_RPC_ENDPOINT = 'http://localhost:8545';
```

Comment:
This needs to match the host and port of test setup phase. We should use either use environment variables or some other methodology.